config/provisioners/natss/100-natss.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: v1
```

Comment:
Can you add a `README` at the root of the `natss` folder, similar like the `in-memory` and `kafka` channel provisioners have it, e.g.:
https://github.com/knative/eventing/tree/master/config/provisioners/kafka

Reply:
feel free to include links to setup for nats(s) as well

Reply:
Done